DataManagementSystem/scripts/dirac-dms-mirror.py

```diff
@@ -0,0 +1,423 @@
+#!/usr/bin/env python
+########################################################################
+# $HeadURL$
+# File :    dirac-dms-mirror
+# Author :  Marko Petric
+########################################################################
+"""
+  Provides basic rsync functionality for DIRAC
+"""
+
+__RCSID__ = "$Id$"
+
+import os
+import sys
+import DIRAC
+from DIRAC.Core.Base import Script
+
+Script.setUsageMessage( '\n'.join( [ __doc__.split( '\n' )[1],
+                                     'Usage:',
+                                     '  %s Source Destination' % Script.scriptName,
+                                     ' ',
+                                     ' e.g.: Download',
+                                     '   %s LFN Path' % Script.scriptName,
```

Comment:
I think it is dangerous to decide upload/download on the number of arguments. There should be a mandatory flag to decide on upload or download